projects/tinkerbell/cluster-api-provider-tinkerbell/Makefile

```diff
@@ -20,7 +20,7 @@ s3-artifacts: create-manifests
 
 .PHONY: create-manifests
 create-manifests: tarballs | ensure-yq $$(ENABLE_DOCKER)
-	@build/create_manifests.sh $(REPO) $(OUTPUT_DIR) $(ARTIFACTS_PATH) $(GIT_TAG) $(IMAGE_REPO) $(IMAGE_TAG)
+	@build/create_manifests.sh $(REPO) $(OUTPUT_DIR) $(ARTIFACTS_PATH) $(GIT_TAG) $(IMAGE_REPO) $(IMAGE_TAG) $(GOLANG_VERSION)
```

Comment:
this is something to look out for elsewhere. ive noticed this kind of "miss" in other projects as well. your solution is correct tho.